### ISSUE TITLE
About CHB-MIT

### ISSUE THREAD

**Issue by 935963004** [ycq091044/BIOT#5]
Dear Chaoqi,
    I am very interested in this work and I find it really meaningful. However, I found the data division of CHB-MIT in the code inconsistent with that in the paper. In the paper, it is stated that patients 20,21 are used for validation, and patients 22,23 are for test while in the preprocessing code, it seems that patients 21,22 are for validation, and patients 23,24 are for test. Which one is the real setting? Moreover, I am working on this dataset and find the training procedure unstable due to the high imbalance of this dataset. Can you provide more details of training this dataset (such as hyperparameters of the focal loss)? It would be appreciated if you can answer my questions. Thanks for your excellent work!

**Comment by ycq091044** [comment]
Thanks for your attention on my paper! 

In the data processing paragraph, I have mentioned that For CHB-MIT (containing 23 patients), we first use patient 1 to 19
for training, 20,21 for validation, and 22,23 for test. Then, we flip the validation and test sets and conduct the experiments again, and we report the average performance on these two settings.

The high imbalance of the dataset can be addressed by (i) when processing the dataset, you could generate more minority class by using overlappings; (ii) using focus loss (alpha=0.8, gamma=0.7 for CHB-MIT).

**Comment by 935963004** [comment]
Thanks a lot for your reply!
